src/state/courses/reducers.js

```diff
@@ -9,7 +9,10 @@ const initialState = {
     list: {
       courses: [],
       isLoadingCourses: true
-    }
+    },
+    // TODO: ver como hacer que no ocupe mucho espacio
```

Comment:
why do you say this?

Reply:
Nevermind, I thought that it could grow a lot. But now that I thought about it, It not going to happen. Comment deleted!